src/todo/ToDoContainer.js

```diff
@@ -9,8 +10,7 @@ function Todo(){
 
     return(
         <div>
-            <input type="text" id="fname" name="fname" placeholder="Input" onChange={(event)=>setInitialText(event.target.value)}></input>
-            <button type="button" onClick={clicked}>Click</button>
+            <InputAndBtn setInitialText={setInitialText} clicked={clicked}/>
             <ListListOfItems list={currentText}/>
```

Comment:
`ListListOfItems` is a pretty confusing name. Since this component is a list of `TodoItem`s why not just name it `TodoList`